.gitignore

```diff
@@ -130,4 +130,6 @@ dmypy.json
 
 # VSCode
 
-.vscode
\ No newline at end of file
+.vscode
```

Comment:
@shanest btw. how do you like visual studio for python code? I'm on linux with PyCharm and love it, but would recommend visual studio for windows users?